MAUI/Calendar/customizations.md

```diff
@@ -0,0 +1,257 @@
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                        View="Month"
+                        Background="PaleGreen">
+            <Calendar:SfCalendar.YearView>
+                <Calendar:CalendarYearView TodayBackground="Pink" DisabledDatesBackground="Grey" LeadingDatesBackground="Green" />
+            </Calendar:SfCalendar.YearView>
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+        CalendarTextStyle textStyle = new CalendarTextStyle()
+        {
+            TextColor = Colors.Black,
+            FontSize = 12,
+        };
+
+        this.Calendar.View = CalendarView.Decade;
+        this.Calendar.MinimumDate = DateTime.Now.AddYears(-1);
+        this.Calendar.MaximumDate = DateTime.Now.AddYears(8);
+        this.Calendar.EnablePastDates = false;
+        this.Calendar.SelectableDayPredicate = (date) =>
+        {
+            if (date.Year == DateTime.Now.AddYears(3).Year)
+            {
+                return false;
+            }
+            return true;
+        };
+
+        this.Calendar.Background = Colors.PaleGreen;
+        this.Calendar.YearView.TextStyle = textStyle;
+        this.Calendar.YearView.TodayBackground = Colors.Pink;
+        this.Calendar.YearView.TodayTextStyle = textStyle;
+        this.Calendar.ShowTrailingAndLeadingDates = true;
+        this.Calendar.YearView.LeadingDatesBackground = Colors.Green;
+        this.Calendar.YearView.LeadingDatesTextStyle = textStyle;
+        this.Calendar.YearView.DisabledDatesBackground = Colors.Grey;
+        this.Calendar.YearView.DisabledDatesTextStyle = textStyle;
+
+{% endhighlight %}
+{% endtabs %}
+
+![decadeview-customization-in-maui-calendar](images/customization/decadeview-customization-in-maui-calendar.png)
+
+>**NOTE**
+* The Background color and text style will be applied based on the following order: selectableDayPredicate dates, disable dates, today date and leading Dates.
+
+## Year View Text format
+You can customize the month format of the `Calendar` by using the `MonthFormat` property in `YearView` class.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                      View="Year">
+            <Calendar:SfCalendar.YearView>
+                <Calendar:CalendarYearView MonthFormat="MMMM" />
+            </Calendar:SfCalendar.YearView>
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.YearView = new CalendarYearView()
+{
+    MonthFormat = "MMMM",
+};
+
+{% endhighlight %}
+{% endtabs %}
+
+![yearview-text-format-in-maui-calendar](images/customization/yearview-text-format-in-maui-calendar.png)
+
+## Selection cell customization
+
+You can customize the Selection cell background and textStyle in Month, Year, Decade and Century view by using the `SfCalendar`.
```

Comment:
You can customize the selection cell background and text style in Month, Year, Decade, and Century view by using the `SfCalendar.`

Reply:
Content changed